cli/cmd/uninstall/uninstall_test.go

```diff
@@ -356,6 +357,7 @@ func getInitializedCommand(t *testing.T) *Command {
 
 	baseCommand := &common.BaseCommand{
 		Log: log,
+		UI:  terminal.NewBasicUI(context.TODO()),
```

Comment:
Added this because this assignment was deferred before I changed the UI to allow writing into any buffer. This just creates a UI that writes to `stdout`.